sql/telemetry_derived/ssl_ratios_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: SSL Ratios
+description: >
+  Percentages of page loads Firefox users have performed that were conducted over SSL.
+owners:
+  - chutten@mozilla.com
```

Comment:
I'm actually not sure about this, but I saw that @chutten is the [owner of the dag in telemetry-airflow](https://github.com/mozilla/telemetry-airflow/blob/831fe84a36347f440ede4f5a90e0bf83d4fa1e1e/dags/ssl_ratios.py#L12). But correct me if it should be someone else.

Reply:
If anyone would like to volunteer themselves, I'm happy to take myself out of the way. But I'm still probably the one people should come to with questions about this dataset, so it's fair. : )